content/news/XXXX-XX-XX-mixxx-2-4-0-features.md

```diff
@@ -0,0 +1,85 @@
+title: Mixxx 2.4 Release
+authors: Jörg Wartenberg
+status: draft
+tags: release, 2.4
+comments: yes
+
+
+We are proud to announce Mixxx 2.4, which will be released NEXT WEEK, a new major version of our free open source DJ software. It took about two years since the last release due to the large number of new features and changes under the hood.
+
+Here you can find some highlights of the Mixxx 2.4 features:
+
+### EngineOS export
+Mixxx 2.4 adds support for exporting crates, playlists and the library to Engine DJ OS devices (Denon and Numark standalone controllers).
+This allows users to prepare their tracks on a laptop using Mixxx, export them on an USB stick and plug it to the standalone controller at your gig.
+
+### Saved Loops
+Mixxx 2.4 adds support for Saved Loops
+If you have already tracks with Saved Loops defined by Serato DJ, Mixxx 2.4 will automatically read them from the track metadata.
+
+<video autoplay loop><source src="/images/news/SavedLoopsFeature.mp4" type='video/mp4'></video>
+
+### Effects
+Mixxx 2.4 introduces Effect Chains, a set of effects with parameters which can be saved and loaded with all it's parameters.
+In the effect units, Mixxx 2.4 visualizes these effect parameter by value, unit and knob position.
+Once you've created an Effect Chain, you can also define individual Super Knob mappings, that allows to use it as Quick-Effect, where you take creative control with a single knob.
+
+### Sync
+Mixxx 2.4 adds support to set an explicit leader for sync, by click on the new button with the crown symbol next to the Sync button.
+This mode is useful when tracks have variable BPM and sync is enabled, all the decks will follow the changing BPM of the sync leader.
+
+### Track Menu
+The track menu in Mixxx 2.4 is extended by several new features:
+- An action to select loaded track in library
+- An action to analyze/re-analyze with variable/constant BPM on a per-track basis
+- An action for looking up track metadata at Discogs, Soundcoud and LastFM
+- An action to remove track files from disk
```

Comment:
```suggestion
- select loaded track in library
- re-/analyze with variable/constant BPM on a per-track basis
- look up track metadata at Discogs, Soundcoud and LastFM
- remove track files from disk
```

Reply:
_ping_

Reply:
I agree, but please capitalize the bullet points.